spyder/app/mainwindow.py

```diff
@@ -332,6 +332,7 @@ def signal_handler(signum, frame=None):
         self.historylog = None
         self.ipyconsole = None
         self.variableexplorer = None
+        self.figureexplorer = None

```

Comment:
Please rename this plugin to `plots`, as we have discussed before.

My plan in beta2 is to remove the `explorer` additional name in all plugins except the Variable Explorer. The thing is in some languages `explorer` generates redundancy in our plugin names because it comes first. For example, in Spanish we have:

* Explorador de Archivos (File Explorer)
* Explorador de Proyectos (Projects Explorer)
etc.

By removing `explorer`, I think users could read our plugin names more easily and that would allow them to switch to them faster too.

Reply:
And we'd also create a common naming scheme with RStudio (another advantage too).